src/components/ReservationsList.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaFacebook, FaTwitter } from 'react-icons/fa';
+import { AiOutlineMail } from 'react-icons/ai';
+import { BiLeftArrow, BiRightArrow } from 'react-icons/bi';
+import { Carousel } from 'react-responsive-carousel';
+import { useLocation } from 'react-router';
+import { fetchreservations } from '../redux/reservation/reservation';
+import styles from '../styles/Home.module.css';
+
+export default function ReservationsList() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const ReservationsList= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪